function-reference/navigateTo.md

```diff
@@ -15,7 +15,7 @@ navigateTo(pageName: string) -> void
 
 | **Argument Name** | **Description** |
 | :--- | :--- |
-| **pageName** |  Page name to which we would like to be transported. It's case sensitive. |
+| **pageNameOrUrl** |  Page name or URL to which we would like to be transported. PageName is case sensitive. |
+| **params** |  Query parameters passed via the URL. Used to share information with destination page. |
```

Comment:
Sorry, I just noticed. Since we are adding the `params` to the Arguments section here, can we also update the `Signature` section to reflect this? Else, the arguments is out of sync with the function signature.

Reply:
Apologies, I didn't noticed it. The `params` are optional, should we use `navigateTo(pageName: string, params?: {}) -> void` or a normal `navigateTo(pageName: string, params: {}) -> void` would be better.

[navigateTo Signature](https://github.com/appsmithorg/appsmith/blob/master/app/client/src/utils/autocomplete/EntityDefinitions.ts#L228)

Reply:
@visibleajay we can use params?: {} and mention in the argument table that it is optional